module/project/templates/validator/v1.js

```diff
@@ -13,9 +13,6 @@ module.exports = (req) => {
             req.checkParams('_id').exists().withMessage("required project template id");
             req.checkQuery('solutionId').exists().withMessage("required solution id");
         },
-        details : function () {
```

Comment:
@VISHNUDAS-tunerlabs please add check to have template id or link in URL, any one is mandatory.